extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-fluid.js

```diff
@@ -62,6 +63,8 @@ const mockPromise = {
   },
 };
 
+window.originalConsoleLog = window.console.log.bind(console.log)
```

Comment:
if you are debugging, the `--verbose` flag might be helpful here.

Reply:
Thanks!! Not familiar with this codebase haha

Reply:
no worries. I usually throw in a `debugger` and then run something like `gulp unit --nobuild --watch --files=<path-to-this-file>` and you can get it to stop where you expect. `nobuild` because I have `gulp` running in another process